extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -80,13 +83,19 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
   const qidoDicomWebClient = staticWado
     ? new StaticWadoClient(qidoConfig)
     : new api.DICOMwebClient(qidoConfig);
-  const wadoDicomWebClient = new api.DICOMwebClient(wadoConfig);
+
+  // const wadoDicomWebClient = new api.DICOMwebClient(wadoConfig);
+  const wadoDicomWebClient = staticWado
+    ? new StaticWadoClient(wadoConfig)
```

Comment:
Just always create the StaticWadoClient, as it falls back to the base client if not required.